docs/source/user-manual/blocks.md

```diff
@@ -0,0 +1,54 @@
+---
+myst:
+  html_meta:
+    "description": "This is the end-User documentation for Plone 6 Volto frontend."
+    "property=og:description": "Volto is the React-based frontend for the Plone CMS. It is the default UI for the Plone 6 release."
+    "property=og:title": "Frontend"
+    "keywords": "Volto, Plone, frontend, React, User manual"
+---
+
+```{todo}
+This page needs content
+```
+
+# Blocks
+Volto features the Pastanaga Editor Engine, allowing you to visually compose a page using blocks. The editor allows you to add, modify, reorder and delete blocks given your requirements. Blocks provide the user the ability to display content in an specific way, although they can also define behavior and have specific features.
+
+## Types of pre-built Blocks
+
+You can access and choose one block out of different types of pre-built blocks by:
+1. clicking on the `+` button provide on the left-side of empty block.
```

Comment:
```suggestion
1.  Click the `+` button on the left-hand side of the empty block.
```

Reply:
Use this screenshot, nested within the list item.

![Screen Shot 2022-12-03 at 2 50 47 PM](https://user-images.githubusercontent.com/102112/205465377-d7dd7c69-bdbc-4702-baee-6ef0aa8102a1.png)